package.json

```diff
@@ -58,7 +58,8 @@
     "lint-staged": "^13.0.3",
     "minimist": "^1.2.6",
     "npm-run-all": "^4.1.3",
-    "prettier": "^2.7.1"
+    "prettier": "^2.7.1",
+    "promise": "^7.3.1"
```

Comment:
Why is native Promise not being used?

Reply:
This package is only used in `TaskQueue-test.js`, which has almost the same test cases as in [`react-native`](https://github.com/facebook/react-native/blob/main/Libraries/Interaction/__tests__/TaskQueue-test.js).
It ensures that [the code](https://github.com/ntdiary/react-native-web/blob/3db9b387a329ea755ebd069e54f028b20b237189/packages/react-native-web/src/exports/InteractionManager/TaskQueue.js#L103-L104) in `task.gen().then()` is called before the corresponding `expect`.
We can change the test cases and then remove this package, it will not affect normal use cases.
```diff
-import Promise from 'promise';

 function expectToBeCalledOnce(fn) {
   expect(fn.mock.calls.length).toBe(1);
@@ -68,8 +67,10 @@ describe('TaskQueue', () => {

     clearTaskQueue(taskQueue);

-    expectToBeCalledOnce(onMoreTasks);
-    expectToBeCalledOnce(task2);
+    return Promise.resolve(() => {
+      expectToBeCalledOnce(onMoreTasks);
+      expectToBeCalledOnce(task2);
+    })
   });

   it('should handle nested simple tasks', () => {
@@ -115,10 +116,12 @@ describe('TaskQueue', () => {

     clearTaskQueue(taskQueue);

-    expectToBeCalledOnce(task1);
-    expectToBeCalledOnce(task2);
-    expectToBeCalledOnce(task3);
-    expectToBeCalledOnce(task4);
+    return Promise.resolve(() => {
+      expectToBeCalledOnce(task1);
+      expectToBeCalledOnce(task2);
+      expectToBeCalledOnce(task3);
+      expectToBeCalledOnce(task4);
+    });
   });
```